include/sound/sof/ext_manifest.h

```diff
@@ -61,6 +61,7 @@ enum sof_ext_man_elem_type {
 	SOF_EXT_MAN_ELEM_WINDOW			= SOF_IPC_EXT_WINDOW,
 	SOF_EXT_MAN_ELEM_CC_VERSION		= SOF_IPC_EXT_CC_INFO,
 	SOF_EXT_MAN_ELEM_DBG_ABI		= SOF_IPC_EXT_USER_ABI_INFO,
+	SOF_EXT_MAN_ELEM_CONFIG_DATA		= 6, /**< ABI3.18 */
```

Comment:
```suggestion
	SOF_EXT_MAN_ELEM_CONFIG_DATA	= 5,    /**< ABI3.17 */
	SOF_EXT_MAN_ELEM_PLATFORM_CONFIG_DATA = 6,  /**< ABI3.18 */
```
https://github.com/thesofproject/sof/blob/f97e9f0de6b76c2b20f3423b57739af8ecf48a42/src/include/kernel/ext_manifest.h#L37-L46

Please align the ABI with SOF FW.

Reply:
done